cmd/cert-checker/main.go

```diff
@@ -51,8 +51,8 @@ type report struct {
 func (r *report) save(directory string) error {
 	filename := path.Join(directory, fmt.Sprintf(
 		"%s-%s-report.json",
-		r.begin.Format(filenameLayout),
 		r.end.Format(filenameLayout),
+		r.begin.Format(filenameLayout),
```

Comment:
Why would the beginning be second? What code is consuming this and can we flip it?

Maybe begin is poorly named?